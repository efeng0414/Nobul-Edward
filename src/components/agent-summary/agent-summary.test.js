/* eslint-disable no-undef */
import React from "react";
import renderer from "react-test-renderer";
import AgentSummary from "./index";

describe("AgentSummary", () => {
  const componentProps = {
    avatarUrl: "demo_avatar_url.jpg",
    firstName: "shil",
    lastName: "last",
    averageRating: 3.1
  };
  const componentJSX = <AgentSummary {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
