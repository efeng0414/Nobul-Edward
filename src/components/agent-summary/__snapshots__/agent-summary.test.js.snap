// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentSummary Component Snapshot should match existing snapshot 1`] = `
<div
  className="agent"
>
  <span
    className="ant-avatar ant-avatar-lg ant-avatar-circle ant-avatar-image"
    style={Object {}}
  >
    <img
      onError={[Function]}
      src="demo_avatar_url.jpg"
    />
  </span>
  <div
    className="agent-summary"
  >
    <div
      className="agent-summary-rating"
    >
      <div
        className="agent-summary-name"
      >
        shil
         
        last
      </div>
    </div>
    <div
      className="agent-summary-brokerage"
    >
      
    </div>
  </div>
</div>
`;
